### PR TITLE
fix: Removes gradient top on default card

### PR DESCRIPTION
**Pull request by christinamegura**
## DESCRIPTION
This PR removes the gradient top overlay on all default cards.

### What does this PR do, or why is it needed?
This PR removes `overlayType` and `overlayColor` on the default card and horizontal default card.

### How do I test this PR?
Use `storybook` to navigate to default cards and horizontal default cards to confirm no gradient.

Simulator Screen Shot - iPhone 11 Pro - 2020-09-25 at 12 39 58

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by christinamegura** [comment]
@vinnyjth, can you get this to pass unit tests? Or help me before Friday? I don't want to go down the test rabbit hole again

File: packages/apollos-ui-kit/src/HorizontalDefaultCard/index.js [hunk 1]
```diff
@@ -44,7 +44,6 @@ const Image = withTheme(
     forceRatio: hasTitleAndSummary ? 2 : 1.5, // forces the placeholder to use the same ratio as above.
     maintainAspectRatio: true,
     overlayColor: theme.colors.black,
```
Comment (vinnyjth) [inline packages/apollos-ui-kit/src/HorizontalDefaultCard/index.js:46]: If we don't use `overlayType`, we probably don't need `overlayColor` :)

File: packages/apollos-ui-kit/src/DefaultCard/index.js [hunk 2]
```diff
@@ -35,7 +35,6 @@ const Image = withTheme(
     maxAspectRatio: 1.78,
     maintainAspectRatio: true,
     overlayColor: theme.colors.black,
-    overlayType: 'gradient-top',
```
Comment (vinnyjth) [inline packages/apollos-ui-kit/src/DefaultCard/index.js:38]: Nice catch, changing this in both files :)

**Review by vinnyjth — CHANGES_REQUESTED** [review]

**Comment by vinnyjth** [comment]
@christinamegura I believe you introduced an unintentional snap change to `AddPrayerScreenConnected`. Not sure how, but if you back it out I believe tests will pass fine.

**Comment by vinnyjth** [comment]
Simulator Screen Shot - iPhone 11 - 2020-09-22 at 14 10 54

@christinamegura I think we'll need a little more work here, the `GradientOverlayImage` defaults to the `default` overlay if you don't specify an `overlayType`. We might have a `none` overlay type, or you'll have to find another way to turn off the overlay. Peeking at the code, I think there's a way ;)

File: packages/apollos-ui-kit/src/HorizontalDefaultCard/index.js (hunk 1 above)
Comment (christinamegura) [reply packages/apollos-ui-kit/src/HorizontalDefaultCard/index.js:46]: I totally had that removed at first, shucks. Changing now, thank ya!

**Comment by vinnyjth** [comment]
@christinamegura When you get the tests passing again, throw the flag on and I'll give this another review :)

**Comment by vinnyjth** [comment]
@christinamegura I think you need to update snaps again, then you should be good to go!

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**